packages/parser/lib/css_parser.mly

```diff
@@ -31,16 +33,20 @@ open Css_types
 %token <string> NTH_FUNCTION
 %token <string> URL
 %token BAD_URL
+%token <string> AT_KEYWORD
 %token <string> AT_MEDIA
 %token <string> AT_KEYFRAMES
 %token <string> AT_RULE
 %token <string> AT_RULE_STATEMENT
-%token <string> HASH
-%token <string> NUMBER
+%token <string * [`ID | `UNRESTRICTED]> HASH
+%token <string> HASH_
+%token <string> NUMBER_
+%token <float> NUMBER
 %token <string> UNICODE_RANGE
 %token <string * string> FLOAT_DIMENSION
-%token <string * string> DIMENSION
-%token <string list> VARIABLE
+%token <float * string> DIMENSION
```

Comment:
I believe we want to store the value of the dimensio as string, so not floating issues appear while parsing